src/pluto/helpers.py

```diff
@@ -17,41 +17,51 @@ def get_output_folder_path(branch: str) -> str:
 
 
 def get_data(branch: str) -> Dict[str, pd.DataFrame]:
-    rv = {}
+    data = {}
     url = f"https://edm-publishing.nyc3.digitaloceanspaces.com/{get_output_folder_path(branch)}"
 
     client = DigitalOceanClient(bucket_name=BUCKET_NAME, repo_name=REPO_NAME)
     kwargs = {"true_values": ["t"], "false_values": ["f"]}
 
-    rv["df_mismatch"] = client.csv_from_DO(
+    data["df_mismatch"] = client.csv_from_DO(
         url=f"{url}/qaqc/qaqc_mismatch.csv", kwargs=kwargs
     )
-    rv["df_null"] = client.csv_from_DO(url=f"{url}/qaqc/qaqc_null.csv", kwargs=kwargs)
-    rv["df_aggregate"] = client.csv_from_DO(
+    data["df_null"] = client.csv_from_DO(url=f"{url}/qaqc/qaqc_null.csv", kwargs=kwargs)
+    data["df_aggregate"] = client.csv_from_DO(
         url=f"{url}/qaqc/qaqc_aggregate.csv", kwargs=kwargs
     )
-    rv["df_expected"] = client.csv_from_DO(
+    data["df_expected"] = client.csv_from_DO(
         url=f"{url}/qaqc/qaqc_expected.csv",
         kwargs={"converters": {"expected": json.loads}} | kwargs,
     )
-    rv["df_outlier"] = client.csv_from_DO(
+    data["df_outlier"] = client.csv_from_DO(
         url=f"{url}/qaqc/qaqc_outlier.csv",
         kwargs={"converters": {"outlier": json.loads}} | kwargs,
     )
 
-    rv = rv | get_changes(client, branch)
+    data = data | get_changes(client, branch)
 
-    rv["source_data_versions"] = client.csv_from_DO(
+    data["source_data_versions"] = client.csv_from_DO(
         url=f"{url}/source_data_versions.csv"
     )
 
-    rv["version_text"] = get_version_text(rv["source_data_versions"])
+    data["version_text"] = get_version_text(data["source_data_versions"])
 
-    return rv
+    # standarzie minor versions strings to be dot notation
+    data_to_standardize = ["df_mismatch", "df_null"]
+    for data_name in data_to_standardize:
+        data[data_name].replace(
+            to_replace={
+                "23v1 - 22v3_1": "23v1 - 22v3.1",
```

Comment:
Any reason to not do this with a regex to make this a bit more general?

Reply:
I think my main reason was that I didn't wanna plan for this to be a persistent issue / method, but instead fix how this data-library archives pluto data to avoid propagating the underscore in the URL

Reply:
made [an issue in data-library](https://github.com/NYCPlanning/db-data-library/issues/379)

Reply:
thankfully no. that "pair string" is already good since it was based on our build of `22v3.1` and not our archiving of it via URL. screenshot below of the strings in the relevant file `qaqc_mismatch.csv`

<img width="161" alt="image" src="https://user-images.githubusercontent.com/7444289/224826670-8aa9992f-5fb8-490b-9acb-2eecea672767.png">